config/open-feature/org/workgroup.yaml

```diff
@@ -32,6 +32,7 @@ approvers:
   - thiyagu06
   - thomaspoignant
   - lukas-reining 
+  - sindhuinti
```

Comment:
Hey @Sindhuinti, please add yourself to the org instead of a spec reviewer. That will allow us to @ mention you and is a prereq to becoming a maintainer. Thanks!

https://github.com/open-feature/community/blob/main/config/open-feature/org.yaml#L29

Reply:
So I have to close this PR @beeme1mr ?

Reply:
No, you can make the change in this PR. You just need to remove your name from this list and add it under the member section in the `org.yaml` file.